erdpy/docker.py

```diff
@@ -0,0 +1,57 @@
+import os
+import logging
+import subprocess
+from pathlib import Path
+from typing import List
+from erdpy.errors import DockerMissingError
+
+
+logger = logging.getLogger("build-with-docker")
+
+
+def is_docker_installed():
+    try:
+        result = subprocess.run(["docker", "--version"], capture_output=True, text=True)
+        output = result.stdout
+    
+        if "Docker version" in output:
+            return True
+        else:
+            raise DockerMissingError()
+    except:
+        logger.error("Something went wrong when checking if docker is installed!")
```

Comment:
I am not sure, isn't this dead code?

Reply:
Because now the function does not raise an error anymore, I expect that if for some reason something goes wrong with the `subprocess.run` the except statement gets executed.